client/game/board/Card.svelte

```diff
@@ -0,0 +1,113 @@
+<script>
+	export let shape, fillStyle, color, number;
+	export let letter = "A";
+	export let selected = false;
+
+	import CardShape from "./CardShape.svelte";
+	import { createEventDispatcher } from "svelte";
+
+	const dispatch = createEventDispatcher();
+	const countToSize = [50, 30, 30];
+	$: humanReadable = number + " " + color + " " + fillStyle + " " + shape + (number == 1 ? "" : "s");
```

Comment:
Yes, it is a [reactive declaration](https://svelte.dev/tutorial/reactive-declarations) and recomputes the value whenever any of the variables on the right change. It might actually not be needed since a card shouldn't change itself during its lifetime.